src/Components/Components/src/PersistentComponentState.cs

```diff
@@ -13,26 +13,36 @@ namespace Microsoft.AspNetCore.Components;
 public class PersistentComponentState
 {
     private IDictionary<string, byte[]>? _existingState;
```

Comment:
Why do we need _existingState anymore?

Reply:
We need `_existingState` for restoring the application state after emitting the comment containing the application state into html page.